persistence/persistence-api/src/main/java/org/cbioportal/persistence/util/CacheEventLogger.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
```

Comment:
I don't think these MSK Copyright still applicable anymore. For the newly created files, we should avoid adding them.